modules/openapi-generator/src/main/resources/go/configuration.mustache

```diff
@@ -26,7 +26,11 @@ var (
 	ContextAccessToken = contextKey("accesstoken")
 
 	// ContextAPIKey takes an APIKey as authentication for the request
-	ContextAPIKey = contextKey("apikey")
+	{{#authMethods}}
+    {{#isApiKey}}
+		Context{{name}} = contextKey("{{name}}")
```

Comment:
Seems like this line is not indented correctly (1 additional tab)

Reply:
And what about making ContextAPIKey a map instead as that's what we do in oher languages?